service/src/main/java/datawave/microservice/authorization/userdetails/ProxiedEntityUserDetailsService.java

```diff
@@ -53,16 +51,6 @@ public UserDetails loadUserDetails(PreAuthenticatedAuthenticationToken token) th
         }
         ProxiedEntityPreauthPrincipal principal = (ProxiedEntityPreauthPrincipal) principalObj;
         
```

Comment:
Removed this code because it's duplicative.  The code path goes through the AllowedCallersFilter (or subclass) before it gets to this point.  Also, we would not have access to the httprequest path at this point to allow calls to the oauth endpoints.

Reply:
Assuming that there are no contexts that would utilize ProxiedEntityUserDetailsService independent from AllowedCallersFilter, then this PR looks good to me